### PR TITLE
Advanced interstitial support

### PR DESCRIPTION
**Pull request by yoalex5**
GitHub issue #218 

1. A new init `init(configId: minWidthPerc: minHeightPerc:)` was added
2. Unit tests were added

The new API looks like

Swift:

```
InterstitialAdUnit(configId: "1001-1", minWidthPerc: 50, minHeightPerc: 70)
```

Objective-C

```
[[InterstitialAdUnit alloc] initWithConfigId:@"1001-1" minWidthPerc:50 minHeightPerc:70];
```

OpenRTB request:
```
request.device.ext.prebid.interstitial: {"minwidthperc": 50, "minheightperc": 70}
```

TODO:
1. Update the doc

### CODE REVIEW

File: src/PrebidMobile/PrebidMobile/AdUnits/Utils.swift [hunk 1]
```diff
@@ -241,6 +241,63 @@ public class Utils: NSObject {
         return gcSize
     }
 
+    func getObjectWithoutEmptyValues(_ dict: [AnyHashable: Any]) -> [AnyHashable: Any]? {
```
Comment (ppuviarasu) [inline src/PrebidMobile/PrebidMobile/AdUnits/Utils.swift]: The utils is becoming a heavy lifter. Do you want to separate out the features?

Comment (yoalex5) [reply src/PrebidMobile/PrebidMobile/AdUnits/Utils.swift]: I will update the PR

**Comment by ppuviarasu** [comment]
This PR has conflicts.

**Comment by yoalex5** [comment]
@ppuviarasu, I have merged master into current branch. Could you please review it ?

**Pull request merged**